Tools/autotest/arduplane.py

```diff
@@ -1723,6 +1723,41 @@ def MainFlight(self):
         self.run_subtest("Mission test",
                          lambda: self.fly_mission("ap1.txt", strict=False))
 
+    def PitotBlockage(self):
+        '''Test detection and isolation of a blocked pitot tube'''
+        self.set_parameter("ARSPD_OPTIONS",15)
+        self.set_parameter("ARSPD_USE",1)
+        self.set_parameter("SIM_WIND_SPD",10)
+        self.set_parameter("SIM_WIND_DIR",0)
+        self.set_parameter("ARSPD_WIND_MAX",15)
+        self.wait_ready_to_arm()
+        self.change_mode("TAKEOFF")
+        self.arm_vehicle()
+        self.delay_sim_time(60)
```

Comment:
Seems a bit arbitrary :-)

Waiting for a specific heading would work well for takeoff mode, so you know the thing's started its post-loiter takeoff.

Reply:
I'll see if I can check roll angle to see when it starts the turn into the loiter which should shorten test time a bit.

Reply:
I removed the delay which fails the pitot tube at start of takeoff which shortens the test and also makes the test more realistic.